backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -361,6 +367,18 @@ public Person addPatient(
     return savedPerson;
   }
 
+  /** This method associates PhoneNumbers with a new patient and ensures there are no duplicates */
+  public List<PhoneNumber> assignPhoneNumbersToPatient(Person person, List<PhoneNumber> incoming) {
```

Comment:
Can you add a new test for this method?

Reply:
✅

Reply:
https://github.com/CDCgov/prime-simplereport/blob/772af8cf3c097024c60cff573f76e8fb339ee4d9/backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java#L304-L304